src/amo/components/FeaturedCollectionCard/index.js

```diff
@@ -0,0 +1,40 @@
+/* @flow */
+import * as React from 'react';
+
+import LandingAddonsCard from 'amo/components/LandingAddonsCard';
+import { INSTALL_SOURCE_FEATURED_COLLECTION } from 'core/constants';
+import type { AddonType } from 'core/types/addons';
+
+export type LandingCardCollectionMetadata = {|
+  footerText: string,
+  header: string,
+  slug: string,
+  username: string,
+|};
+
+type Props = {|
+  addons: Array<AddonType>,
+  className: string,
+  collectionMetadata: LandingCardCollectionMetadata,
+  loading: boolean,
+|};
+
+export default class FeaturedCollectionCard extends React.Component<Props> {
+  render() {
```

Comment:
Because we use shallow rendering, this function won't ever get executed and you'll see a reduced code coverage score. I'd like to see a simple test for it to make sure it executes without throwing. Since none of the props are dynamic, the test won't have to do much.

Reply:
Well, huh. The codecov report says it's covered. Let me know if I'm missing something.

Reply:
Is this because the test uses `shallow` rather than `shallowUntilTarget`? Would it make sense to change it to the latter? If not, how exactly do I write a test targeting the `render` method?

Reply:
Oh, sorry, I guess I saw your first comment and not your second. Does this mean nothing needs to be done for this?

Reply:
Sorry, ignore me. I didn't even notice `TestFeaturedCollectionCard.js` for some reason.